bundle-workflow/README.md

```diff
@@ -77,3 +101,22 @@ Signing step (to sign all artifacts):
 ```bash
 ./bundle_workflow/sign.sh artifacts/manifest.yml
 ```
+
+### Plugin Onboarding
+
+1. Update a [manifest](/manifests) for a particular release to include your plugin. We require your plugin name, repository url, and git ref
+that should be used. For unreleased versions this should be a branch in your repository.  Once a release is cut, these refs will be updated
+to build from a tag or specific commit hash.
+
+2. Create a build.sh if you have specific requirements that are not covered by the default script and commit it to your repository. It
+ can live at the root our under scripts/build.sh.
```

Comment:
- Let's swap these and make a recommendation that script/build.sh is preferred, to avoid noise at the root.
- Link the default script.